frontend/common/types/requests.ts

```diff
@@ -191,10 +242,10 @@ export type Req = {
   }
   getLaunchDarklyProjectImport: { project_id: string; import_id: string }
   getLaunchDarklyProjectsImport: { project_id: string }
-  getUserWithRoles: { org_id: string; user_id: string }
-  deleteUserWihRole: { org_id: string; user_id: string; role_id: string }
-  getGroupWithRole: { org_id: string; group_id: string }
-  deleteGroupWithRole: { org_id: string; group_id: string; role_id: string }
+  getUserWithRoles: { org_id: number; user_id: number }
+  deleteUserWithRole: { org_id: number; user_id: number; role_id: number }
+  getGroupWithRole: { org_id: number; group_id: number }
+  deleteGroupWithRole: { org_id: number; group_id: number; role_id: number }
```

Comment:
I'm surprised by the switch from `organisation_id` to `org_id` here.

Reply:
Yes I think there's more I'd refactor including this, but I've already fixed/changed a lot of types in this PR already